src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random();
```

Comment:
Consider moving the `Random` instance to be a class-level field instead of creating a new instance every time the method is called. This will improve performance as you will reuse the same `Random` instance.